changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
+= 2.8.5 - 2024-xx-xx =
+* Fix   - Adding Product Add-Ons on order edit page is stucked when automated tax is enabled.
```

Comment:
```suggestion
* Fix   - Adding Product Add-Ons on order edit page is broken when automated tax is enabled.
```

Reply:
Thanks for the suggestion @PanosSynetos , I'll change the wording with your suggestion.